apidoc/Titanium/UI/iPhone/iPhone.yml

```diff
@@ -216,42 +216,26 @@ properties:
         since: "3.1.3"
         notes: Use [Titanium.UI.Window.statusBarStyle](Titanium.UI.Window.statusBarStyle) instead.
         removed: "6.0.0"
----
-name: hideStatusBarParams
-summary: Dictionary of options for the <Titanium.UI.iPhone.hideStatusBar> method.
-since: 2.0.0
-platforms: [iphone, ipad]
 
-properties:
-  - name: animated
-    summary: Determines whether to animate the dialog as it is hidden.
-    type: Boolean
-    
-  - name: animationStyle
-    summary: Style of the animation.
-    description: |
-        One of the status bar animation style constants 
-        <Titanium.UI.iPhone.StatusBar.ANIMATION_STYLE_NONE>, 
-        <Titanium.UI.iPhone.StatusBar.ANIMATION_STYLE_SLIDE>, 
-        or <Titanium.UI.iPhone.StatusBar.ANIMATION_STYLE_FADE>.
-    type: Number
+  - name: hideStatusBarParams
```

Comment:
Why do we do this? The `----` is meant to be used  for dictionaries that need to be passed to other API's. They are no properties / methods.